### ISSUE TITLE
Coleco>Skip BIOS intro works inconsistently and can introduce bugs

### ISSUE THREAD

**Issue by GoogleCodeExporter** [TASVideos/BizHawk#205]
```
What steps will reproduce the problem?
1. Load the game Mr. Do! into ColecoHawk.
2. Check "Skip BIOS intro" if it is not already checked.
3. Reboot the core to apply the checkbox toggle. 
4. The BIOS intro will not be skipped.

What is the expected output? What do you see instead?
I expect to see a short black screen followed by the classic ColecoVision blue 
"select a game mode" screen. Instead, I see the ~11 second long BIOS intro 
("COLECOVISION PRESENTS UNIVERSAL'S 'MR. DO!'") first.

What version of the product are you using? On what operating system?
BizHawk 1.7.1. Windows 7 Pro 64-bit.

Please provide any additional information below.
The BIOS intro is successfully skipped for the game Burgertime.

For the game Venture, BizHawk skips the BIOS intro but the game freezes a few 
frames after entering a room for the first time no matter what you do, 
rendering the game uncompletable.
```

Original issue reported on code.google.com by `coolkirb...@gmail.com` on 9 Jul 2014 at 2:31

**Comment by alyosha-tas** [comment]
I had some time to look at this today. A few things to note.

1, Bios skipping is not always applicable, it is handled in the game DB with a 'no skip' flag if it cannot be skipped. Mr Do! had this flag, so bios couldn't be skipped. This wasn't apparent to the user, so I added a little note along with the check box.

2. Venture seems like it should have the 'no skip' flag, but doesn't. I tried to resolve it but it seems you just can't skip it with method used in ColecoHawk. So I added the falg to the DB.

3. I don't really understand why the method used by ColecoHawk to skip the BIOS should be allowed at all, since it modifies the game ROM to do so, I think an explanation would be helpful here, maybe I'll look into it.
